lantern_test.go

```diff
@@ -25,6 +25,7 @@ func (c testUserConfig) AfterStart()              {}
 func (c testUserConfig) BandwidthUpdate(int, int) {}
 func (c testUserConfig) ConfigUpdate(bool)        {}
 func (c testUserConfig) ShowSurvey(survey string) {}
+func (c testUserConfig) SetStaging(bool)          {}
```

Comment:
Drive-by fix for a compilation issue.  This type needs to implement the UserConfig interface.  As that interface is implemented by this package, this could not be something broken by a dependency update.  This is why I think the tests were already broken.